packages/react/src/themes/teams/components/Chat/chatMessageVariables.ts

```diff
@@ -4,13 +4,13 @@ export interface ChatMessageVariables {
   actionMenuBoxShadow: string
   actionMenuPositionRight: string
   actionMenuPositionTop: string
-  width: string
```

Comment:
@kuzhelov we're removing variables, doesn't this mean it's a breaking change?

Reply:
actually, this is a change introduced to the consumer's code (without any change provided to Stardust's approach to styling) - thus I wouldn't expect increment of Stardust version because of that

Reply:
makes sense